AutomatedTesting/Levels/WhiteBox/EmptyLevel/EmptyLevel.prefab

```diff
@@ -0,0 +1,118 @@
+{
+    "ContainerEntity": {
```

Comment:
Just curious. Why do whitebox tests need their own empty level? Did you try just using the 'base' prefab level that we use for other tests?

Reply:
I just checked and it is actually NOT empty in Whitebox/EmptyLevel. There is still `DefaultLevelSetup` entity in the level.

Reply:
Can you add a follow up task to rename this to 'DefaultSetupLevel' or something to include that its not empty?

Reply:
Sure I will.

Reply:
Follow-up: #7503